Gemfile.lock

```diff
@@ -204,7 +204,6 @@ GEM
 
 PLATFORMS
   x86_64-darwin-20
-  x86_64-linux
```

Comment:
Let's leave LINUX option for vagrant users.

Reply:
ok